mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala

```diff
@@ -82,6 +82,8 @@ object Tool extends LazyLogging {
    *   the exit code; as usual, 0 means success.
    */
   def run(args: Array[String]): Int = {
+    // Configure the silent logger first. Otherwise, Apache Commons spills a lot of text to the console.
+    new LogbackConfigurator(None, None).configureDefaultContext()
```

Comment:
Why do we need this? Why wasn't it needed before?

Reply:
I think we did not have any behavior in the tests that would require error messages when parsing the input configuration. In other words, potential configuration parsing issues were silently ignored.